### ISSUE TITLE
[QST] From psychology to Machine Learning

### ISSUE THREAD

**Issue by bjpietrzak** [MaartenGr/MaartenGr#2]
Hi 😄

I read your articles on Medium.

It's incredible how you've switched from psychology to computer science and can showcase such expertise with ML.

I have a question regarding this subject. My background is similar to yours. I studied cognitive science, but currently finishing 3rd year of CS and working as a ML engineer.

I presume psychology doesn't pay much attention to STEM subjects like mathematics or physics. That's why I wanted to ask you.

QUESTION:
How did you manage to change your career? Did you self-study? If so, what resources did you use? What did you focus on?

I still feel like I'm walking blindly in my  ML journey, especially when it comes to mathematics. Any form of advice or response would mean a lot to me.

**Comment by MaartenGr** [comment]
Hi, thank you for your kind words! 

> I presume psychology doesn't pay much attention to STEM subjects like mathematics or physics. That's why I wanted to ask you.

Psychology tends to focus on statistics quite a bit more than you would expect. Many of the things (research) psychologists do are statistical experiments do evaluate and prove concepts. So I had some background in statistics before transitioning into ML. I should note though that I did not have any programming experience before starting and always did statistics in SPSS using its GUI. So very very basic!

> How did you manage to change your career? Did you self-study? If so, what resources did you use? What did you focus on?

This is a very difficult question to answer in just a few lines but I'll try my best. 

I started out with a master's degree in Industrial Psychology and a master's degree in Clinical Psychology, so I already had some experience with research. After that, I pursued a master's degree in Data Science which by itself did not fully prepare me for ML. In parallel, I wrote blog posts, studied ML from the classics (e.g., An Introduction to Statistical Learning), wrote open-source projects, and generally did as much as possible to get up to speed. 

The exact things I did are not that helpful to you since the field has changed significantly by then. The one thing that I can advise here is to start doing projects as much as possible. Make sure that they are as fun as possible. The reason for that is there is a difference between learning the theory and truly understanding it. I could learn all manners of theoretical underpinnings but that does not mean I would be able to apply them in practice. 

I wrote a couple of articles on this subject that might be interesting to check out ([1](https://www.maartengrootendorst.com/blog/ambiguous/), [2](https://www.maartengrootendorst.com/blog/portfolio/), [3](https://www.maartengrootendorst.com/blog/truths/)). I also created a [post](https://www.maartengrootendorst.com/blog/transition/) about this very subject but it is a bit outdated (2020!). 

> I still feel like I'm walking blindly in my ML journey, especially when it comes to mathematics. Any form of advice or response would mean a lot to me.

I had the same issue when starting out! This is an incredibly fast-pacing field that even I find difficult to keep up with.  Although I never really had a mentor, I can definitely advise looking for someone to find guidance with. Generally, it would be at a place you already work or have an internship with. 

Either way, finding guidance and making sure that whatever you do is fun helps tremendously. 

Hopefully, this helps a bit!
